apl-core/src/main/java/com/apollocurrency/aplwallet/apl/exchange/service/DexService.java

```diff
@@ -92,4 +94,22 @@ public boolean widthraw(String account,String secretPhrase,BigDecimal amount,Str
     }
 
 
+    public void closeOverdueOrders(){
+        List<DexOffer> offers = dexOfferDao.getOverdueOrders(epochTime.getEpochTime());
+
+        for (DexOffer offer : offers) {
+            try {
+                Account account = Account.getAccount(offer.getAccountId());
+                offer.setStatus(OfferStatus.CLOSED);
+
+                dexOfferTable.insert(offer);
```

Comment:
Better to check here, that multiple sql updates are executing during one transaction